api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/innsending/InnsendingRoute.kt

```diff
@@ -67,7 +70,16 @@ fun Route.innsendingRoute(
                                     logger.info(it)
                                     sikkerLogger.info("$it og request:\n$json")
                                 }
-                            }.fromJson(Innsending.serializer())
+                            }.let { json ->
+                                runCatching {
+                                    json.fromJson(SkjemaInntektsmelding.serializer()).convert(
+                                        sykmeldingsperioder = emptyList(),
+                                        aarsakInnsending = AarsakInnsending.Ny,
```

Comment:
Prøver å forstå dette. Når er det egentlig AarsakInnsending blir til Endring her?

Reply:
Greit å verifisere i backend sikkert. Men er det mulig å se i payloaden fra frontend at det er endring?
Må man ikke gi en begrunnelse for endring etc.

Reply:
Det feltet blir bestemt av om man sender fra en kvitteringsside eller ei, så brukeren bestemmer ikke noe over feltet. Begrunnelse for endring er for inntekt.

Reply:
Nå har jeg fikset det. I starten setter man `AarsakInnsendig.Ny` uansett, men så endres det i https://github.com/navikt/helsearbeidsgiver-inntektsmelding/pull/619/files#diff-a305504fd5988ecbc6a069c88045df4d7077a7966c13974403ae8aea298d2a89R191.